src/main/webapp/js/userMapData.json

```diff
@@ -857,6 +904,7 @@
       "California State University Long Beach",
       "California State University Northridge",
       "California State University San Marcos",
+      "California State University, Stanislaus",
```

Comment:
It is a real thing `actually`[https://www.csustan.edu/] and this is why I included it. As for the commas, instead of removing that I entered a comma in all the previous ones since there was already in inconsistency with "East Bay" (which had comma) and the others. I did enter a comma in all those lines because this is how I found them when I googled for them.